src/System.Runtime.Intrinsics.X86/src/Configurations.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+    </BuildConfigurations>
+    <BuildConfigurations>
```

Comment:
Remove this whoel BuildConfigurations section, All you need in this property group is:

```xml
  <PropertyGroup>
    <BuildConfigurations>
      netcoreapp;
    </BuildConfigurations>
   </PropertyGroup>
```

Reply:
@mellinoe @joperezr Sorry, this comment should be here.

I tried, but an error occurred: 
```
D:\workspace\corefx\buildvertical.targets(162,5): error : Could not find a configuration for ProjectReference 'D:\workspace\corefx\\external\runtime\runti
me.depproj' from configurations 'netcoreapp-Windows_NT;netcoreapp-Unix;uap;uapaot;mono' when building 'System.Runtime.Intrinsics.X86' for configuration 'n
etcoreapp-Release' . [D:\workspace\corefx\src\System.Runtime.Intrinsics.X86\src\System.Runtime.Intrinsics.X86.csproj] Command execution failed with exit code 1.
```

Reply:
@mellinoe @joperezr ping?

Reply:
Given that you are directly referencing System.Ptivate.CoreLib which is OS specific you will need to have netcoreapp-Windows_NT and netcoreapp-Unix. So the second set of BuildConfigurations. You can remove the first set.

Reply:
@weshaggard Do you mean that I should remove `netcoreapp;` and just use `netcoreapp-Windows_NT;` and `netcoreapp-Unix;` in src/Configuration.props? If yes, how to change .csproj and ref/Configuration.props?

Reply:
Yes that is what I mean. For the ref/Configuration.props it should still just be netcoreapp (i.e. there isn't no OS specific differences between the APIs).  The csproj's should match the configurations for the particular project.

Reply:
Thank you, will do.